api/controllers/environment_test.go

```diff
@@ -22,8 +22,9 @@ func TestCreateEnvironment(t *testing.T) {
 
 	req := models.CreateEnvironmentRequest{
 		EnvironmentName: "env",
-		InstanceSize:    "m3.medium",
-		MinClusterCount: 1,
+		InstanceType:    "m3.medium",
```

Comment:
[nit] we're using a lot of different instance types/sizes throughout this changeset. I see `t2`, `m3` --- the size differences make sense in tests and such, but maybe we should consider just sticking with `t2` for type.

Reply:
the only non-test value is `DefaultEnvironmentInstanceType = "t2.small"` in config/defaults.go

Reply:
right, but why even keep `m3` around if we don't use it for a real default anywhere? Still a [nit] regardless.